packages/utils-tests-helpers/src/hint-runner.ts

```diff
@@ -244,6 +249,14 @@ export const testHint = (hintId: string, hintTests: HintTest[], configs: { [key:
         } catch (e) {
             console.error(e);
 
+            if (server) {
+                server.stop();
```

Comment:
Should this also be an `await`?
```suggestion
                await server.stop();
```

Reply:
Yup